packages/remix/src/ssr/rootAuthLoader.ts

```diff
@@ -67,6 +78,8 @@ export const rootAuthLoader: RootAuthLoader = async (
     }
   }
 
-  // if the user returned a plain object, create an empty response and inject requestState and handlerResult
-  return injectRequestStateIntoResponse(new Response(JSON.stringify(handlerResult)), requestState);
+  // if the return value of the user's handler is null or a plain object, create an empty response to inject Clerk's state into
+  const responseBody = JSON.stringify(handlerResult ?? {});
```

Comment:
by this point `handlerResult` should be an object or null, so the use of `??` feels meaningful here.